### ISSUE TITLE
Contribution guide

### ISSUE THREAD

**Issue by ganyicz** [larapeeps/larapeeps.com#21]
Multiple people have asked about adding new peeps to the list, we should communicate clearly on the website that the list is open-source and the criteria for adding (= to fit at least one group)

**Comment by CodyPChristian** [comment]
I agree and how about a spot for people like myself who builds with Laravel daily for clients? In case anyone is looking for a curated 'approved/vetted' list they can reach out to? Thoughts?
